blocks/cards/cards.css

```diff
@@ -0,0 +1,32 @@
+.cards > ul {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(15rem, 1fr));
+    grid-gap: 16px;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    font-size: var(--body-font-size-xs);
+}
+
+.cards > ul > li {
+    border: 1px solid var(--overlay-background-color);
+    border-radius: 10px;
+    transition: all 0.5 ease-out; 
+    box-shadow: 1px 1px 5px #00000020;
+}
+
+.cards > ul > li:hover {
+    box-shadow: 2px 2px 10px #00000040;
+    transform: translate(-1px, -1px)
+}
```

Comment:
I think is is potentially confusing... this makes me think that the card is clickable.

Reply:
true true... fixed